readthedocs/embed/views.py

```diff
@@ -135,12 +134,10 @@ def get(self, request):
 def do_embed(*, project, version, doc=None, path=None, section=None, url=None):
     """Get the embed response from a document section."""
     if not url:
-        external = version.type == EXTERNAL
```

Comment:
Do we not need this anymore?

Reply:
We get the type of the version from the version object itself